### ISSUE TITLE
System.register using fully qualified `file:` path when bundling

### ISSUE THREAD

**Issue by twalker** [jspm/jspm-cli#511]
After installing jspm (v0.13.4) today, `bundle` seems to be registering a fully qualified path.

```javascript
// yesterday:
System.register("src/app", ["src/models/somemodel"], function (_export) { /*...*/ }
```

```javascript
// today:
System.register("file:/projects/jspm-6to5-example/public/js/src/models/somemodel", [], function (_export) { /*...*/ }
```

Can be reproduced here:
https://github.com/twalker/jspm-6to5-example

**Comment by guybedford** [comment]
Thanks, this is fixed in SystemJS builder 0.8.2.

**Comment by twalker** [comment]
Joyfully verified.
